pkg/controller/common/rollout/workloads/cloneset_controller.go

```diff
@@ -138,6 +131,10 @@ func (c *CloneSetController) RolloutOneBatchPods(ctx context.Context) (bool, err
 	newPodTarget := c.calculateNewPodTarget(int(cloneSetSize))
 	// set the Partition as the desired number of pods in old revisions.
 	clonePatch := client.MergeFrom(c.cloneSet.DeepCopyObject())
+
+	if err := c.handler.rolloutOneBatchPods(ctx, c.cloneSet, newPodTarget); err != nil {
+		return false, err
+	}
 	c.cloneSet.Spec.UpdateStrategy.Partition = &intstr.IntOrString{Type: intstr.Int,
```

Comment:
why is this still here? It seems that the scale part shouldn't touch the partition.

Reply:
OK. I'll delete it.